Dockerfile

```diff
@@ -45,7 +48,7 @@ RUN apk add netcat-openbsd
 WORKDIR /app
 COPY ./wait-for ./
 ENV PATH /app:$PATH
-COPY --from=build /build/lakefs /build/lakectl ./
+COPY --from=build /build/lakefs /build/lakefs.cover /build/lakectl ./
```

Comment:
Are we sure that we want our official docker image to contain another binary which sole purpose is to be run in Nessie?
I would much rather we build a dedicated image for Nessie that will contain just the cover instead of adding it to our published image. @nopcoder WDYT?